Backend/dhall-configs/dev/rider-app.dhall

```diff
@@ -185,16 +185,13 @@ let jobInfoMapx =
       ]
 
 let cacConfig =
-        { host = "http://localhost:8080"
-        , interval = 10
-        , tenants = [ "dev", "test" ]
-        , retryConnection = False
-        }
-      : { host : Text
-        , interval : Natural
-        , tenants : List Text
-        , retryConnection : Bool
-        }
+      { host = "http://localhost:8080"
+      , interval = 10
+      , tenant = "dev"
```

Comment:
Make sure connection will always happen to this even if it is missing in cacTenants.